warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

```diff
@@ -66,9 +66,30 @@ def open_browser(self, browser_name='firefox', webdriver_remote_url=False,
         browser_name = browser_name
         browser = self._make_browser(browser_name, desired_capabilities,
                                      profile_dir, webdriver_remote_url,
-                                     binary=binary, gecko_path=gecko_path)
+                                    binary=binary, gecko_path=gecko_path)
+        print_info("The Selenium Webdriver version is '{0}'".format(webdriver.__version__))
+        if browser:
+            browser_detail_dict = self.get_browser_details(browser)
+            for details, value in browser_detail_dict.items():
+                print_info("The Browser '{0}' {1} is '{2}'".format(browser_name, details, value))
         return browser
 
+    def get_browser_details(self, browser):
+        """ Return the Browser details dict. The Dict can be updated with additional information
+        about the Browser
+
+            Arguments :
+                       browser = The browser instance
+
+            Return :
+                     browser_detail_dict = A Dictionary containing details of the browser instance
+                     such as version.
+
+        """
+        browser_detail_dict = {}
+        browser_detail_dict['version'] = browser.capabilities['version']
```

Comment:
Not account for the new firefox capabilities key "browserVersion" here

Reply:
This is particular to Selenium version 3.x. This is found and logic has arrived. 
You could find the logic on how I have handled multiple versions here. https://github.com/warriorframework/warriorframework/blob/WAR-10074/warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

Reply:
Hi @brettleehari 
In the future please make sure to include the necessary fix like this when merging/releasing new feature, thank you